modules/android/pages/vector-search.adoc

```diff
@@ -97,13 +150,19 @@ Choosing Centroids in vector search involves trade-offs that can impact clusteri
 
 The general guideline for the amount of Centroids is approximately the square root of the number of documents.
 
+=== About Probes
```

Comment:
This is a little jarring--I don't know why you re mentioning probes....start with a statement to let user know what these are

Reply:
This is now addressed with clearer explanation surrounding probes.